src/main/java/org/forgerock/openam/auth/nodes/EmailNotifyNode.java

```diff
@@ -66,18 +68,19 @@ public interface Config {
         default String smtpHostName() { return "localhost"; }
         @Attribute(order = 600)
         default String smtpHostPort() { return "25"; }
-        @Attribute(order = 700)
-        default String smtpUserName() { return "username"; }
+        @Attribute(order = 700, requiredValue = false)
+        String smtpUserName();
         @Attribute(order = 800)
-        default String smtpUserPassword() { return "secret123"; }
+        @Password
```

Comment:
Should this also have requiredValue = false then?

Reply:
This doesn't seem to have an effect on password types in the UI because the value is masked anyway.. It is not marked as a required field.  Added for uniformity.